apis/clientlogapi/clientlogapi.yaml

```diff
@@ -0,0 +1,481 @@
+        required:
+         - pdata
+         - context
+         - logs
+        type: object
+        properties:
+          pdata:
+            $ref: '#/components/schemas/Pdata'
+          context:
+           $ref: '#/components/schemas/Context'
+          logs:
+           type: array
+           items:
+             $ref: '#/components/schemas/Logs'
+           description: logs
+      Pdata:
+       title: Pdata
+       required:
+         - id
+         - ver
+         - pid
+       properties:
+         id: 
+           type: string
+           description: Producer id
+         ver:
+           type: string
+           description: Error Event version
+         pid:
+           type: string
+       example:
+         id: contentPlayer
+         ver: '1.0'
+         pid: sunbird.portal
+      Logs:
+        title: Logs
+        required:
+          - log
+          - ts
+        properties:
+          id:
+            type: string
+            description: Unique id of log
+          ts:
+            type: number
+            description: Log timestamp
+          log:
+            type: string
+            description: exception of the log event
+          appver:
+            type: string
+          pageid:
+            type: string
+        example:
+          id: '13123-123123'
+          ts: "1560346371"
+          log: 'Exception in thread \"main\" java.lang.NullPointerException'
+          appver: '1.0'
+          pageid: 'player'
+      Context:
+        title: Context
+        required:
+         - did
+        properties:
+          did:
+           type: string
+           description: Unique device identifier
+          dspec:
+            $ref: '#/components/schemas/Dspec'
+          extras:
+           type: object
+      Dspec:
+        title: Dspec
+        properties: 
+          os: 
+           type: string 
+          make: 
+            type: string
+          mem:
+            type: integer
+          idisk:
+            type: string
+          edisk: 
+            type: string 
+          scrn:
+            type: string
+          camera:
+            type: string 
+          cpu: 
+            type: string
+          sims: 
+            type: integer
+          uaspec: 
+            $ref: '#/components/schemas/Uaspec'
+        example:
+          os: mac
+          make: ''
+          mem: 0
+          idisk: ''
+          edisk: ''
+          scrn: ''
+          camera: ''
+          cpu: ''
+          sims: 0
+          uaspec: '#/components/schemas/Uaspec/example'
+      Uaspec:
+        title: Uaspec
+        properties: 
+          agent: 
+           type: string 
+          ver: 
+            type: string
+          system:
+            type: integer
+          platform:
+            type: string
+          raw: 
+            type: string
+        example:
+          agent: ''
+          ver: ''
+          system: ''
+          platform: ''
+          raw: ''
+         
+      ClientLogAPI-200OK:
```

Comment:
What is ClientLogAPI-200OK? Please name the title appropriately instead of using Http error codes.